internal/api/v1beta1/user.go

```diff
@@ -392,3 +401,8 @@ func transformUserToPB(usr user.User) (shieldv1beta1.User, error) {
 		UpdatedAt: timestamppb.New(usr.UpdatedAt),
 	}, nil
 }
+
+func isValidEmail(email string) bool {
```

Comment:
The logic seems scattered around.
I suggest to have 1 specific function to normalize the user struct. Inside it, it should include the `checking email format` and `lowercasing the email`.

Reply:
Yeah its scattered. 
I think ww should keep validations in handler and business logic in service layer. So invalid email checking is correctly placed in handler.

Shall we conisder normalizing email as a business logic or validation/sanitaion check?

Reply:
Invalid email checking should also be part of the business here I believe. It is part of user domain.